src/components/heading/heading.style.js

```diff
@@ -64,4 +61,71 @@ StyledHeadingIcon.defaultProps = {
   theme: baseTheme,
 };
 
-export { StyledHeadingIcon, StyledHeading };
+const StyledSubHeader = styled.div`
+  margin-top: 5px;
+`;
+
+StyledSubHeader.defaultProps = {
+  theme: baseTheme,
+};
+
+const StyledHeader = styled.div`
+  position: relative;
+`;
+
+StyledHeader.defaultProps = {
+  theme: baseTheme,
```

Comment:
This isn't needed because `StyledHeader` doesn't use the theme

Reply:
removed